services/telemetry_v2/ds_vm.go

```diff
@@ -0,0 +1,84 @@
+package telemetry_v2
+
+import (
+	"context"
+	pmmv1 "github.com/percona-platform/saas/gen/telemetry/events/pmm"
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+type dsVm struct {
```

Comment:
what does dsVM stands for?

Reply:
dataSourceVictoriaMetrics?
let's use full name for struct type

Reply:
> dataSourceVictoriaMetrics?
yes

> let's use full name for struct type
yeah, totally make sense